logging/jboss/src/test/java/io/quarkus/ts/logging/jboss/DefaultMinLogLevelIT.java

```diff
@@ -9,6 +23,11 @@
 @QuarkusScenario
 public class DefaultMinLogLevelIT {
 
+    private static final String LOG_FILE_NAME = "quarkus.log";
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM");
+    private static final String EXPECTED_LOG_MESSAGE = "Example log message";
+    private final String LOG_DIR = "target/" + this.getClass().getSimpleName() + "/app";
```

Comment:
this won't do, please use app service folder on `app` static property

Reply:
I can change it, np

Reply:
changed